justfile

```diff
@@ -53,17 +53,19 @@ create-devnet:
         --args-file network_params.yaml \
         --enclave {{enclave}}
 
+l1-rpc-url := `kurtosis service inspect devnet el-1-geth-lighthouse | grep -- ' rpc: ' | sed 's/.*-> //'`
```

Comment:
Is this change needed? I originally moved these into the `generate-fixture` script so that these `kurtosis` calls would only need to be made just-in-time, instead of at the start of any `just` call

Reply:
Oh yeah I think this is because `shell()` doesn't work for me for some reason. What's the set up that can make `shell()` work?

Reply:
Discussed offline - it appears that `shell()` is a relatively new feature of `just`, and required an upgrade